enterprise/coderd/coderd.go

```diff
@@ -0,0 +1,30 @@
+package coderd
+
+import (
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/coderd"
+	"github.com/coder/coder/coderd/rbac"
+)
+
+func NewEnterprise(options *coderd.Options) *coderd.API {
```

Comment:
A lot of `coderd.Options` initialization occurs at the beginning of `coderd/coderd.go` right now. eg. this package will have to reinitialize `PrometheusRegistry` and such as well. I'm not convinced this is a problem though, as 2x duplication isn't really a cause for concern IMO.